app/templates/template.html

```diff
@@ -18,17 +18,17 @@
         {% block headIcons %}
             <link rel="icon"
                   sizes="48x48"
-                  href="{{ assetPath | default('/assets', true) }}/images/favicon.ico">
+                  href="{{ assetPath | default('/assets', true) }}/image/favicon.ico">
             <link rel="icon"
                   sizes="any"
-                  href="{{ assetPath | default('/assets', true) }}/images/favicon.svg"
+                  href="{{ assetPath | default('/assets', true) }}/image/favicon.svg"
                   type="image/svg+xml">
             <link rel="mask-icon"
                   href="{{ assetPath | default('/assets', true) }}/images/govuk-icon-mask.svg"
```

Comment:
Does this path not also need changing?

Reply:
@anthonyhashemi theoretically yes, although wasn't 100% sure what i should replace the mask-icon with, couldn't find a mask-icon on the official TNA website, I could ask Terry and see if we could perhaps reuse the ```favicon.svg``` for this purpose

Reply:
oh I see, yeah see what Terry thinks

Reply:
@anthonyhashemi should be sorted now 🤝